eqcorrscan/utils/clustering.py

```diff
@@ -13,24 +13,25 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-import numpy as np
-import warnings
-import matplotlib.pyplot as plt
 import os
-
+import warnings
 from multiprocessing import Pool, cpu_count
-from scipy.spatial.distance import squareform
-from scipy.cluster.hierarchy import linkage, dendrogram, fcluster
-from obspy.signal.cross_correlation import xcorr
+
+import matplotlib.pyplot as plt
+import numpy as np
```

Comment:
Is there any reason for moving the numpy import lower? I was having them at the top to follow ObsPy, but didn't have a better reason.

Reply:
I use pycharm as an IDE. After writing some code I usually apply use the code reformat function to clean up all the nit-picky pep-8 stuff. As a consequence, it sorts the imports into the following 3 categories, separated by a blank line, outlined by pep8's section on imports:

    standard library imports
    related third party imports
    local application/library specific imports

since numpy is not part of the standard library it gets put into the second category and alphabetized. I can put it back but this is the most pep8 compliant.

Reply:
Sweet, no, I'm not that picky about the imports, and if this is more pep8 friendly then it seems like a good way to go - I use pycharm too, should probably go through the whole codebase at some point and run the code reformat (lots of things in pycharm I don't know about!).